docker-bake.hcl

```diff
@@ -0,0 +1,128 @@
+variable "GO_VERSION" {
+  default = "1.17.0"
+}
+variable "BIN_OUT" {
+  default = "./bin"
+}
+variable "RELEASE_OUT" {
+  default = "./release-out"
+}
+
+// Special target: https://github.com/docker/metadata-action#bake-definition
+target "docker-metadata-action" {
```

Comment:
No changes are needed. We already discussed it probably but in this context, a name like "_ci_config", "_ci_inputs" would make more sense. Doesn't need to be builtin if metadata-action allows override in github yaml.

Reply:
> We already discussed it probably but in this context, a name like "_ci_config", "_ci_inputs" would make more sense.

Named `meta-helper` like in binfmt repo.

> Doesn't need to be builtin if metadata-action allows override in github yaml.

Wdym? Remove this target and let the metadata-action handles the inheritance in `image` and `image-cross` targets with smth like `--set image.inherits.push(meta-helper)`? `push()` would add a value to the existing `inherits` slice.

Reply:
Just meant `bake-target: _ci_inputs` . Didn't know metadata-action already allowed setting the name.